client/tokenized-express-checkout/compatibility/wc-product-page.js

```diff
@@ -37,14 +37,16 @@ addFilter(
 	'wcpay.express-checkout.cart-add-item',
 	'automattic/wcpay/express-checkout',
 	( productData ) => {
-		const $variationInformation = jQuery( '.single_variation_wrap' );
-		if ( ! $variationInformation.length ) {
+		const variationInformation = document.querySelector(
```

Comment:
I moved away from the jQuery selectors and switched to vanilla JS selectors, so that testing would be more reliable and easier (without having to mock jQuery)